backend/apps/mentorship/admin.py

```
@@ -0,0 +1,107 @@
+
+from django.contrib import admin
+
+from apps.mentorship.models.enrollment import Enrollment
+from apps.mentorship.models.mentee import Mentee
+from apps.mentorship.models.mentor import Mentor
+from apps.mentorship.models.module import Module
+from apps.mentorship.models.program import Program
+from apps.mentorship.models.programmodule import ProgramModule
+
+
+class EnrollmentAdmin(admin.ModelAdmin):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)